src/dds/qos.rs

```diff
@@ -69,21 +70,8 @@ pub struct QosPolicyBuilder {
 }
 
 impl QosPolicyBuilder {
-  pub const fn new() -> QosPolicyBuilder {
```

Comment:
potentially controversial...

the derived `Default` is not `const`. so this is a trade off between performance and readability